### PR TITLE
documentation/DEMO-Added-link-to-codesandbox

### PR DESCRIPTION
**Pull request by elartix**
[https://codesandbox.io/s/redux-saga-controller031-sjc5r](https://codesandbox.io/s/redux-saga-controller031-sjc5r)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
 
 # redux-saga-controller
 
+<p align="center">
```
Comment (ichabanivan) [inline README.md]: Can you use [markdown syntaxes](https://www.markdownguide.org/basic-syntax/) instead of HTML?

**Review by ichabanivan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (elartix) [reply README.md]: githubs markdown doesn't support any alignment

Comment (githubbrightgrove) [reply README.md:4]: We don't need to support alignment, it is not so good mix HTML and markdown

Comment (ichabanivan) [reply README.md]: We don't need to support alignment, it is not so good mix HTML and markdown

**Pull request merged**